src/test/java/com/garannodou/role/controller/GoogleAuthRequestTest.java

```diff
@@ -0,0 +1,190 @@
+package com.garannodou.role.controller;
+
+import com.garannodou.role.controller.GoogleAuthRequest.GoogleAuth;
+import com.garannodou.role.controller.GoogleAuthRequest.GoogleBasicProfile;
+import org.junit.Test;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.is;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+public class GoogleAuthRequestTest {
```

Comment:
I think it is better to add these tests in the UserControllerTest since we are testing input validation. Also these tests could maybe be parameterized